presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java

```diff
@@ -73,6 +76,8 @@
     private final DataSize outputDataSize;
     private final long outputPositions;
 
+    private final List<OperatorStats> operatorSummaries;
```

Comment:
Is this going to include stats for all operators in all tasks? It can get quite large if that's the case.